src/main/java/com/scalar/db/storage/cosmos/MapVisitor.java

```diff
@@ -102,7 +101,7 @@ public void visit(BlobValue value) {
         .get()
         .ifPresent(
             b -> {
-              values.put(value.getName(), new String(value.get().get(), StandardCharsets.UTF_8));
+              values.put(value.getName(), (ByteBuffer) ByteBuffer.allocate(b.length).put(b).flip());
```

Comment:
Just a question.
So, encoding to base64-based string is done by Cosmos DB but decoding should be done by users?

If so, what is the trigger of the automatic encoding? Is it done in case of byte array?
I'm just curious.

Reply:
A byte array is automatically encoded to Base64 string in Azure Cosmos SDK.
But Cosmos DB always returns the encoded string because Cosmos DB doesn't know the actual type when it's written.
So, users should decode it.

Reply:
Got it. Thank you! (It's a pretty weird spec though...)